static/src/js/components/Map/GranuleGridLayerExtended.js

```diff
@@ -821,7 +821,11 @@ export class GranuleGridLayerExtended extends L.GridLayer {
         this._onEdscStickygranule({ granule: null })
       } else {
         const [granule] = defaultGranules.filter((g) => g.id === focusedGranuleId)
```

Comment:
this line could be rewritten:
```js
const granule = defaultGranules.find((g) => g.id === focusedGranuleId)
```

Not required, but just in case you wanted to do it while you are editing nearby code.